nemo/core/classes/modelPT.py

```diff
@@ -941,7 +941,7 @@ def test_dataloader(self):
 
         return self._test_dl
 
-    def on_validation_epoch_end(self) -> Optional[Dict[str, Dict[str, torch.Tensor]]]:
+    def on_validation_epoch_end(self, sync_metrics: bool = False) -> Optional[Dict[str, Dict[str, torch.Tensor]]]:
```

Comment:
Don't change defaults in modelPT. Modify them in ASR model at best.

Reply:
Can we make the `sync_metrics` variable inheritable so that we can control this variable in child classes?
Speaker diarization models also need this to be `sync_dist=False` but currently no way to control this in child classes.

Reply:
@titu1994 
How can we avoid changing this part - without overriding this function with huge amount of duplicated code ? This is a question out of ignorance.

Reply:
@titu1994 modelPT behavior is unchanged (it was already `False`), we need to expose a way to enable the sync in inheriting classes